Add overview of default local.php (#1)
beetofly/user-documentation: docs/configuration/local_php_overview.rst

Local.php fields overview

> #########

Reviewer: This needs to be the full underline length of the title.

> Most but not all Mautic configuration can be done from the user interface.

Reviewer: You need to use active tense here! Also need to capitalise User Interface.

Maybe:

```suggestion
Most but not all Mautic configuration is managed in the User Interface.
```

> Either way, the settings live in the local.php file.

Reviewer: Let's make it clear that we're talking about a filename.

```suggestion
Either way, the settings live in the ``local.php`` file.
```

> Here is the list of the local.php parameters for mautic 4.4.6:

Reviewer: ```suggestion
Here are the ``local.php`` parameters for Mautic 4.4.6:
```

.. list-table::
   :widths: 100 100 100
   :header-rows: 1

   * - Parameter
     - Value
     - Info
   * - allowed_extensions

>      - default: An indexed array with the values csv, doc, docx, epub, gif, jpg, jpeg, mpg, mpeg, mp3, odt, odp, ods, pdf, png, ppt, pptx, tif, tiff, txt, xls, xlsx, wav

Reviewer: Let's put these into code markup.

```suggestion
     - default: an indexed array with the values ``csv``, ``doc``, ``docx``, ``epub``, ``gif``, ``jpg``, ``jpeg``, ``mpg``, ``mpeg``, ``mp3``, ``odt``, ``odp``, ``ods``, ``pdf``, ``png``, ``ppt``, ``pptx``, ``tif``, ``tiff``, ``txt``, ``xls``, ``xlsx``, ``wav``
```

>      - The "Allowed file extensions" parameter from the Mautic configuration settings (Theme settings): https://mautic-documentation.readthedocs.io/en/latest/configuration/settings.html

Reviewer: We need to use internal linking here.

```suggestion
     - The "Allowed file extensions" parameter from the Mautic configuration settings Theme settings: :ref:`Theme settings`
```

>    * - anonymize_ip

Reviewer: Let's code-wrap these values so it's clear that it's a setting variable.

```suggestion
   * - ``anonymize_ip``
```

     - default: 0 (for 'No')

>      - The "Anonymize IP" parameter from the Mautic configuration settings (Tracking settings): https://mautic-documentation.readthedocs.io/en/latest/configuration/settings.html

Reviewer: ```suggestion
     - The "Anonymize IP" parameter from the Mautic configuration tracking settings: :ref:`Tracking settings`
```

   * - api_batch_max_limit
     - default: 200
     -
   * - api_enable_basic_auth
     - default: false

>      - The "Enable HTTP basic auth?" parameter from the Mautic configuration settings (API settings): https://mautic-documentation.readthedocs.io/en/latest/configuration/settings.html

Reviewer: ```suggestion
     - The "Enable HTTP basic auth?" parameter from the Mautic configuration API settings: :ref:`API settings`
```

   * - api_enabled

>      - default: 0 (for 'No')

Reviewer: ```suggestion
     - default: 0 - for 'No'
```

>      - The "API enabled?" parameter from the Mautic configuration settings (API settings): https://mautic-documentation.readthedocs.io/en/latest/configuration/settings.html

Reviewer: ```suggestion
     - The "API enabled?" parameter from the Mautic configuration API settings: :ref:`API settings`
```

   * - api_oauth2_access_token_lifetime

>      - default: 60 (minutes)

Reviewer: ```suggestion
     - default: 60 - minutes
```

>      - The "Access token lifetime (in minutes)" parameter from the Mautic configuration settings (API settings): https://mautic-documentation.readthedocs.io/en/latest/configuration/settings.html

Reviewer: ```suggestion
     - The "Access token lifetime in minutes" parameter from the Mautic configuration API settings: :ref:`API settings`
```

   * - api_oauth2_refresh_token_lifetime

>      - default: 14 (days)

Reviewer: ```suggestion
     - default: 14 days
```
